### PR TITLE
Annotations

### PR DESCRIPTION
**Pull request by alexanderbates**
* Meta annotate annotations
* Lock neurons
* Update neurons

### CODE REVIEW

**Comment by jefferis** [comment]
I have removed the `update_*` functions to their own PR as they are logically a bit different and I want to think about them.

**Comment by jefferis** [comment]
Hi @alexanderbates would you be able to update your checkout to latest state of this branch and implement something for @mmc46 #137? Thanks!

**Comment by alexanderbates** [comment]
@mmc46 I have added the following to this PR:

`catmaid_get_meta_annotations("v14")
catmaid_query_meta_annotations("CATMAID instance")`

**Comment by jefferis** [comment]
OK @alexanderbates thanks for these additions! I'll take it from here.

File: R/catmaid_annotations.R [hunk 1]
```diff
@@ -280,10 +274,7 @@ possibly.numeric <- function(x) {
 #' Lock or unlock a CATMAID neuron reconstruction
 #'
 #' @description  Lock or unlock a CATMAID neuron reconstruction by adding or removing a 'locked' annotation to a set of skeleton IDs (skids). A locked neuron cannot be edited until it is unlocked.
-#' @param skids the skeleton IDs neurons you wish to lock / unlock
-#' @param pid project id. Defaults to 1
-#' @param conn CATMAID connection object, see ?catmaid_login for details
-#' @param ... methods passed to catmaid_fetch
+#' @inheritParams read.neuron.catmaid
```
Comment (jefferis) [inline R/catmaid_annotations.R]: @alexanderbates FYI you can save a bit of typing by inheriting the docs for these 4 arguments from `read.neuron(s).catmaid` (they are jointly documented so both `skid` and `skids` are defined there).

**Pull request merged**